Tests/Unit/EmogrifierTest.php

```diff
@@ -680,6 +680,9 @@ public function cssDeclarationWhitespaceDroppingDataProvider()
             'space before trailing semicolon' => [' color:#000 ;', 'color: #000;'],
             'space after trailing semicolon' => [' color:#000; ', 'color: #000;'],
             'space after property value, no trailing semicolon' => [' color:#000; ', 'color: #000;'],
+            'nl before property value, trailing semicolon' => ["\ncolor:#222; ", 'color: #222;'],
+            'nl before colon and after value, trailing semicolon' => ["color\n:#333\n; ", 'color: #333;'],
+            'nl after colon, trailing semicolon' => ["color:\n#333; ", 'color: #333;'],
```

Comment:
Could you please change the "nl" to "NL" or (even better) "newline" in the (new) descriptions?